OptimizelySDK/ProjectConfig.cs

```diff
@@ -280,6 +296,24 @@ public static ProjectConfig Create(string content, ILogger logger, IErrorHandler
             return config;
         }
 
+        private static ProjectConfig GetConfig(string configData)
+        {
+            if (configData == null)
+                throw new ConfigParseException("Unable to parse null datafile.");
+
+            if (string.IsNullOrEmpty(configData))
+                throw new ConfigParseException("Unable to parse empty datafile.");
+
+            var config = JsonConvert.DeserializeObject<ProjectConfig>(configData);
+
+            if (SupportedVersions.TrueForAll((obj) => !(((int)obj).ToString() == config.Version)))
```

Comment:
Please don't use the generic variable name `obj`. Instead go for `supportVersion`